src/lib/setup.js

```diff
@@ -173,6 +173,17 @@ export default function(app) {
     )
   )
 
+  // Add CSRF to the cookie and remove it from the page. This will allows the
+  // caching on the html.
+  app.use((req, res, next) => {
+    const csrf = res.locals.sd.CSRF_TOKEN
```

Comment:
This seems unused.

Reply:
It definitely is, nice catch. I feel like this is usually caught by the linter.

Reply:
You'll need to change to `.ts` or add `// @ts-check` to the top of the file to get linting here.